src/Messages/Interactions/TSMessage.m

```diff
@@ -132,6 +133,10 @@ - (nullable instancetype)initWithCoder:(NSCoder *)coder
     }
 
     _schemaVersion = OWSMessageSchemaVersion;
+
+    // We _DO NOT_ set _receivedAt_ in this constructor.  We don't want to
+    // set the receivedAt time for old messages in the data store.
```

Comment:
Yeah, that makes sense. I think when I wrote that comment I was confusing `receivedAt` with `receivedAtDate`. Disregard...